README.md

```diff
@@ -77,31 +86,52 @@ We could simply treat it as the tree
+### Finding a Node
 
-JavaScript exposes a few ways of finding DOM nodes more or less directly, courtesy of the `document` object.
+JavaScript exposes a few ways of finding DOM nodes more or less directly,
+courtesy of the `document` object.
 
 #### `document.getElementById()`
 
-This method provides the quickest access to a node, but it requires that we know something very specific about it — its `id`. Since IDs must be unique, this method only returns one element. (If you have two elements with the same ID, this method returns the first one — keep your IDs unique!) Given the following DOM tree
+This method provides the quickest access to a node, but it requires that we know
+something very specific about it — its `id`. Since IDs must be unique, this
+method only returns one element. (If you have two elements with the same ID,
+this method returns the first one — keep your IDs unique!) Given the following
+DOM tree
 
 ``` html
 <div>
   <h5 id="greeting">Hello!</h5>
 </div>
 ```
 
-we could find the `h5` element with `document.getElementById('greeting')`. Notice how the `id` that we pass to `getElementById` is identical to the `id` in `<h5 id="greeting">`. We can assign properties to HTML nodes (or elements) simply by including them between the `<>` tags at the start of the element (so not in the `</h5>` tag, in this case).
+we could find the `h5` element with `document.getElementById('greeting')`.
+Notice how the `id` that we pass to `getElementById` is identical to the `id` in
+`<h5 id="greeting">`. We can assign properties to HTML nodes (or elements)
+simply by including them between the `<>` tags at the start of the element (so
+not in the `</h5>` tag, in this case).
 
 **Try it out!**
 
-Open up your web inspector (command+option+j on OS X) and find an element on the page — make note of its `id`. Then open up your console, type `document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy DOM node. Try changing a few of its properties!
+Open up your web inspector (command+option+j on OS X) and find an element on the
+page — make note of its `id`. Then open up your console, type
+`document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy
+DOM node. Try changing a few of its properties!
 
 #### `document.getElementsByClassName()`
 
-This method, as its name implies, finds elements by their `className`. Unlike `id`, `className` does not need to be unique; as such, this method returns an HTMLCollection (basically a list of DOM nodes — note that it is _not_ an array, even though it has a `length` property) of all the elements with the given class. You can iterate over an HTMLCollection with a simple `for` loop.
+This method, as its name implies, finds elements by their `className`. Unlike
+`id`, `className` does not need to be unique; as such, this method returns an
```

Comment:
❗️ ❗️ 

OK, we still don't have _property_ in their vocabulary or _looping_ yet. Do we need to introduce that idea here lightly. If so, some screenshot of the console where you get length probably makes sense.

If we're going to teach JS here that we don't cover until later (eg vars and loops) maybe we should include some verbiage with "Also we're going to toss some JavaScript programming ideas in here.."

In the interest of an utterly modular curriculum, we may not want to open that front, the goal is still the DOM is a tree whose leaves and branches can be targeted.

Reply:
I'm of two minds here. While I want to cut everything too specific (there's a bit about an array that seems too advanced), at the same time I think this might be a good way to gently, organically introduce some of the programming terms they're going to see in the near future. They're not being asked to come up with any of this code on their own, just copy it and see what it does. It actually could be a good way to visualize how loops work beforehand, so later when they get to the official concept, they'll have a context already in mind.